src/status_im/browser/core.cljs

```diff
@@ -82,10 +82,17 @@
   (let [history-host (http/url-host (try (nth history history-index) (catch js/Error _)))]
     (assoc browser :unsafe? (dependencies/phishing-detect history-host))))
 
-(defn resolve-ens-multihash-callback [hex]
+(defn resolve-ens-content-callback [hex]
   (let [hash (when hex (multihash/base58 (multihash/create :sha2-256 (subs hex 2))))]
     (if (and hash (not= hash resolver/default-hash))
-      (re-frame/dispatch [:browser.callback/resolve-ens-multihash-success hash])
+      (re-frame/dispatch [:browser.callback/resolve-ens-multihash-success "01" hash])
```

Comment:
01 ?

Reply:
this is backwards support for content, we supported only ipfs for content field , https://github.com/ensdomains/multicodec/blob/master/table.csv#L3

Reply:
Cool!